other/limit-configmap-for-sa/limit_configmap_for_sa.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: limit-configmap-for-sa
+  annotations:
+    policies.kyverno.io/title: Limit ConfigMap to ServiceAccounts for a User
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.6.0
+    kyverno.io/kubernetes-version: "1.20-1.23"
```

Comment:
Did you personally test this with these four versions? If so, that's great. If not, this would need to be done for the annotation's value to be written as such.

Reply:
Yes, I did. :)